source/grid/grid_in.cc

```diff
@@ -495,6 +499,35 @@ GridIn<dim, spacedim>::read_vtk(std::istream &in)
 }
 
 
+
+template <int dim, int spacedim>
+void
+GridIn<dim, spacedim>::read_vtu(std::istream &in)
+{
+  namespace pt = boost::property_tree;
+  pt::ptree tree;
+  pt::read_xml(in, tree);
+  auto section = tree.get_optional<std::string>("VTKFile.dealiiData");
+
+  AssertThrow(section,
+              ExcMessage(
+                "While reading a VTU file, failed to find dealiiData section. "
+                "Notice that we can only read grid files in .vtu format that "
+                "were created by the deal.II library, using a call to "
+                "GridOut::write_vtu(), where the flag "
+                "GridOutFlags::Vtu::serialize_triangulation is set to true."));
+
+  std::cout << "Encoded: " << *section << std::endl;
```

Comment:
There shouldn't be any output to `stdout` by default. :slightly_smiling_face:

Reply:
Whoops. 😂

Reply:
Whoops. 😂